DQM/L1TMonitor/src/L1TStage2EMTF.cc

```diff
@@ -999,6 +1047,7 @@ void L1TStage2EMTF::analyze(const edm::Event& e, const edm::EventSetup& c) {
     int modeRPC = Track->Mode_RPC();
     int singleMuQuality = 12;
     int singleMuPT = 22;
+    int singleMuUPT = 10;  //Lucas Faria de Sa Tucker June 28 2023
```

Comment:
Why integers the two pts?
In addition, please make them constexpr (including quality), and move them outside the loop